packages/market-matcher-core/src/CertificateService.ts

```diff
@@ -0,0 +1,111 @@
+import { Demand, PurchasableCertificate } from '@energyweb/market';
+import { Configuration, Unit } from '@energyweb/utils-general';
+import { inject, injectable } from 'tsyringe';
+import * as Winston from 'winston';
+import { TransactionReceipt } from 'web3-core'; // eslint-disable-line import/no-unresolved
```

Comment:
Please remove:
```
// eslint-disable-line import/no-unresolved
```

It should not be used with `web3` imports anymore. If there are errors please install:
```
yarn add web3-core -W
```
to market-matcher-core